src/components/TodoList/TodoList.tsx

```diff
@@ -1,208 +1,85 @@
 /* eslint-disable max-len */
 import React from 'react';
+import classNames from 'classnames';
+import { Todo } from '../../types/Todo';
+import { useAppDispatch, useAppSelector } from '../../app/hooks';
+import { actions as actionsCurrent } from '../../features/currentTodo';
```

Comment:
```suggestion
import { actions as currentTodoActions } from '../../features/currentTodo';
```
sounds better and clearer